superflore/generators/ebuild/run.py

```diff
@@ -82,29 +90,34 @@ def main():
         help='location of the Git repo',
         type=str
     )
+    parser.add_argument(
+        '--only',
+        help='generate only the specified package',
+        type=str
```

Comment:
I'd recommend supporting mutiple listed packages. You can use `nargs='+'` https://docs.python.org/3/library/argparse.html#nargs

Note you'll slightly need to change the logic https://github.com/ros-infrastructure/superflore/pull/76/files#diff-5a7657ae13022a44815efc96706d2233R147 to iterate over the multiple arguments.

Reply:
I was wondering how one would go about this. ;) Thanks for the resources. I'll definitely add this in.